products/veeam-backup-and-replication.md

```diff
@@ -12,84 +12,92 @@ releaseColumn: false
 releaseDateColumn: true
 eolColumn: End of support
 
+# Release dates (RTM) can be found on https://www.veeam.com/kb2680.
+# EOL dates can be found on https://www.veeam.com/product-lifecycle.html.
 releases:
--   releaseCycle: "12.0"
-    releaseDate: 2023-01-30
-    support: true
+-   releaseCycle: "12 GA"
```

Comment:
It's not a good idea to indicate GA/RTM in the `releaseCycle`. `releaseCycle` are used as the name of the version in the API so it must be predictable.

I propose to remove GA/RTM from the `releaseCycle` and just say in the description that the GA date is used as the release date, except for 5.0 because it was not provided. WDYT @adrianwells ?

Reply:
OK. Makes sense. Updated.